src/CMakeLists.txt

```diff
@@ -842,6 +854,18 @@ add_subdirectory(compressor)
 
 add_subdirectory(tools)
 
+# dmClock
+
+add_subdirectory(dmclock) # after gmock
+add_dependencies(tests dmclock-tests dmclock-data-struct-tests)
+
+if(WITH_TESTS)
+  install(PROGRAMS
+    ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/dmclock-tests
+    ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/dmclock-data-struct-tests
+    DESTINATION bin)
+endif(WITH_TESTS)
```

Comment:
although this [1] appears to be building the dmclock tests

[1] https://github.com/ivancich/ceph-fork/blob/wip-bring-in-dmclock-p2-gtest/src/dmclock/CMakeLists.txt#L19-L25